docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -117,6 +145,21 @@ CREATE TABLE student (id INT, name STRING)
 CREATE TABLE student (id INT,name STRING)
     ROW FORMAT DELIMITED FIELDS TERMINATED BY ','
     STORED AS TEXTFILE;
+    
+--Use complex datatype
```

Comment:
Could you split the example into multiple ones, e.g., `use complex datatype`, `Use the native SerDe`, `Use the custom SerDe`, ...?

Reply:
sure, will update later